source/config.go

```diff
@@ -35,6 +41,23 @@ type Config struct {
 }
 
 // Validate executes manual validations beyond what is defined in struct tags.
-func (c Config) Validate() error {
-	return c.Config.Validate()
+func (c *Config) Validate(ctx context.Context) error {
+	err := c.Config.Validate()
+	if err != nil {
+		return err
+	}
+	// validate and set the topics.
+	if len(c.Topic) == 0 && len(c.Topics) == 0 {
+		return fmt.Errorf("required parameter missing: %q", "topics")
+	}
+	if len(c.Topic) > 0 && len(c.Topics) > 0 {
+		return fmt.Errorf(`can't provide both "topic" and "topics" params, "topic" will be deprecated soon, please only provide the "topics" parameter instead`)
+	}
+	if len(c.Topic) > 0 && len(c.Topics) == 0 {
+		sdk.Logger(ctx).Warn().Msg(`"topic" parameter will be deprecated soon, please use "topics" instead.`)
+		// add the topic value to the topics slice.
+		c.Topics = make([]string, 1)
+		c.Topics[0] = c.Topic
```

Comment:
could we maybe actually respect all topics that were provided via `topic`?

e.g.: 

```suggestion
		c.Topics = make([]string, len(c.Topic))
		c.Topics = c.Topic
```

Reply:
`c.Topic` is just a single `string`, what do you mean by respect "all" topics? 🤔

Reply:
@lovromazgon oh, thought this was coming from yesterday's changes in which we had it as a slice (see https://github.com/ConduitIO/conduit-connector-kafka/pull/131/commits/3f258cad16df581013a0ec539b0785989ef6b22b#diff-2f3155aea603bde2c43d7e87f9fd14baf61f282ae3df1873fd70dd6456abc472R27). Didn't check today again that `Topic` had came back to its original type. Apologies for the confusion! @maha-hajja